src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/IsEmpty.cs

```diff
@@ -19,7 +19,7 @@ internal sealed class IsEmptyFunction : BuiltinFunction
         public override bool SupportsParamCoercion => false;
 
         public IsEmptyFunction()
-            : base("IsEmpty", TexlStrings.AboutIsEmpty, FunctionCategories.Table | FunctionCategories.Information, DType.Boolean, 0, 1, 1)
+            : base("IsEmpty", TexlStrings.AboutIsEmpty, FunctionCategories.Table | FunctionCategories.Information, DType.Boolean, 0, 1, 1, DType.EmptyTable)
```

Comment:
>DType.EmptyTable [](http://example.com/codeflow?start=139&length=16)

Will you be writing a converter on paclient side for this change?

Reply:
Yes